tests/resources/test_query_transformation_request.py

```diff
@@ -45,14 +45,51 @@ def test_query_all(self, mocker, mock_rabbit_adaptor):
         assert response.json == [{'request_id': '123'}, {'request_id': '456'}]
         mock_transform_request_read_all.assert_called()
 
-    def test_query_single_request(self, mocker, mock_rabbit_adaptor):
+    def test_query_single_request_no_object_store(self, mocker, mock_rabbit_adaptor):
         import servicex
         mock_transform_request_read = mocker.patch.object(
             servicex.models.TransformRequest,
             'return_request',
             return_value=self._generate_transform_request())
 
-        client = self._test_client(rabbit_adaptor=mock_rabbit_adaptor)
+        local_config = {
+            'OBJECT_STORE_ENABLED': False
+        }
+
+        client = self._test_client(additional_config=local_config,
+                                   rabbit_adaptor=mock_rabbit_adaptor)
+        response = client.get('/servicex/transformation/1234')
+        assert response.status_code == 200
+        print(response.json)
+        assert response.json == {'request_id': 'BR549', 'did': '123-456-789',
+                                 'columns': 'electron.eta(), muon.pt()',
+                                 'selection': None,
+                                 'tree-name': "Events",
+                                 'image': 'ssl-hep/foo:latest', 'chunk-size': 1000,
+                                 'workers': 42, 'result-destination': 'kafka',
```

Comment:
Your destination here is kakfka. I'm not saying this is bad, but since this is basically a new test, perhaps we should pair it with object store to keep it consistent for the future?